### PR TITLE
Added documentation with example usage

### PR DESCRIPTION
**Pull request by ofarukdogancay**
Added documentation with example usage for quick start

@moray95 @cantaylancapraz

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    auto-startup: false
+    job-store-type: jdbc
+    jdbc:
+      initialize-schema: never
+    properties:
+      org.quartz.scheduler.instanceId: AUTO
+      org.quartz.jobStore.dataSource: quartz_ds
+      org.quartz.jobStore.driverDelegateClass: org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
+      org.quartz.jobStore.isClustered: true
+      org.quartz.jobStore.tablePrefix: qrtz_
```
Comment (can-arikan) [inline README.md:35]: we may change it to `<SERVICE_NAME>_qrtz_` hence it doesn't appear enough which may cause confusion if users try to create multiple micro-services in the same database

File: README.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+      org.quartz.dataSource.quartz_ds.provider: hikaricp
+      org.quartz.dataSource.quartz_ds.driver: org.postgresql.Driver
+      org.quartz.dataSource.quartz_ds.URL: ${spring.datasource.url}
+      org.quartz.dataSource.quartz_ds.user: ${spring.datasource.username}
+      org.quartz.dataSource.quartz_ds.password: ${spring.datasource.password}
+
+simplyquartz:
+  enabled: true
+  packagesToScan: com.example
```
Comment (can-arikan) [inline README.md:44]: do we still need this

File: README.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+Note: `@DisallowConcurrentExecution` belongs to quartz library, and `@QuartzSchedule` belongs to simply quartz library.
+`@DisallowConcurrentExecution` is used to prevent the job from running concurrently. If you don't want to prevent
+concurrent execution, you can remove it. This annotation supports clustered mod too and will prevent concurrent
+execution across the cluster, if clustered mod is enabled and instances have unique ids.
+
+Jobs that extend `TimedJob` will automatically record the execution time and count metrics using Micrometer.
+Simply Quartz will create metrics like this:
+
+name: "quartz_scheduled_job"\
```
Comment (can-arikan) [inline README.md:98]: we may remove back slashes

**Review by can-arikan — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ofarukdogancay) [reply README.md:44]: Yes, in large projects it shouldn't scan all packages

File: README.md (hunk 3 above)
Comment (ofarukdogancay) [reply README.md:98]: New line character olarak geçiyor

File: README.md (hunk 1 above)
Comment (ofarukdogancay) [reply README.md:35]: The default of quartz is qrtz_ , the property is shown for capability.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
+Simply Quartz
```
Comment (moray95) [inline README.md]: Burayı # başlık yapalım

File: README.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+@QuartzSchedule(cron = "0/5 * * * * ?")
+@DisallowConcurrentExecution
+class SampleJob : TimedJob() {
+    override fun executeTimed(context: JobExecutionContext?) {
+        println("Hello World!")
+    }
+}
+```
+
+Note: `@DisallowConcurrentExecution` belongs to quartz library, and `@QuartzSchedule` belongs to simply quartz library.
```
Comment (moray95) [inline README.md]: quartz -> Quartz
simply quartz -> SimplyQuartz

**Pull request merged**